### PR TITLE
Tweaks and fixes

### PR DESCRIPTION
**Pull request by dmohs**
Mainly fixes https://broadinstitute.atlassian.net/browse/GAWB-440.

Reviewer: Please suggest fixes for the TODO items in the second commit.
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Make sure liquibase is updated if appropriate
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala [hunk 1]
```diff
@@ -58,7 +61,24 @@ class SlickDataSource(val databaseConfig: DatabaseConfig[JdbcProfile])(implicit
   }
 
   def initWithLiquibase(liquibaseChangeLog: String) = {
-    val dbConnection = database.source.createConnection()
+    val dbConnection = try {
+      database.source.createConnection()
+    } catch {
+      case e: SQLTimeoutException =>
+        // TODO(dmohs): Throwables is not in our build.sbt.
```
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala]: what is this TODO?

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala [hunk 2]
```diff
@@ -58,7 +61,24 @@ class SlickDataSource(val databaseConfig: DatabaseConfig[JdbcProfile])(implicit
...
+        val isCertProblem = Throwables.getRootCause(e).isInstanceOf[SunCertPathBuilderException]
+        if (isCertProblem) {
+          val k = "javax.net.ssl.keyStore"
+          if (System.getProperty(k) == null) {
+            // TODO(dmohs): Log warning.
+            println("***")
+            println("*** The system property '" + k + "' is null. This is likely the cause of the")
+            println("*** database connection failure.")
+            println("***")
```
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala]: all this should be log messages, not println

Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala]: make class SlickDataSource extend LazyLogging then you can do logger.error(...)

File: build.sbt [hunk 3]
```diff
@@ -8,7 +8,8 @@ version       := "0.1"
 
 scalaVersion  := "2.11.7"
 
-scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8")
+scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8",
+  "-Xmax-classfile-name", "100")
```
Comment (jmthibault79) [inline build.sbt]: I'm hesitant to include this without first figuring out why only your computer needs this change.

**Comment by jmthibault79** [comment]
@dmohs over to you to address comments

File: build.sbt (hunk 3 above)
Comment (jmthibault79) [inline build.sbt]: @dmohs This change is now on develop.  Please rebase.

**Comment by dvoet** [comment]
:+1:

**Pull request merged**